spec/features/job_seekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'A job seeker can subscribe to a job alert' do
+  context 'A job seeker' do
+    scenario 'can access the new subscription page when search criteria have been specified' do
+      expect { visit(new_subscription_path) }.to raise_error(ActionController::ParameterMissing)
```

Comment:
We are going directly to the page here. I presume that's because another story will come along and add the start page for the user after they finish a search?

Reply:
The first page is still missing as it won't be implemented until this card https://trello.com/c/OFbNOs2j/660-124-email-alerts-a-visitor-can-subscribe-to-a-search

However, I thought that that page was left in the design as it might be needed later on, and not for implementing now.